### PR TITLE
US End of Year landing page

### PR DESCRIPTION
**Pull request by jlieb10**
## Why are you doing this?
New landing page for the new US moment

## Screenshots
Landing page:
landing page

Thank you 1:
thank you 1

Thank you 2:
thank you 2

### CODE REVIEW

File: support-frontend/assets/pages/contributions-landing/components/ContributionFormContainer.jsx [hunk 1]
```diff
@@ -96,12 +108,20 @@ function withProps(props: PropTypes) {
     props.onThirdPartyPaymentAuthorised(paymentAuthorisation);
   };
 
-  const landingPageCopy = props.isReturningContributor && props.landingPageCopyReturningSinglesTestVariant === 'returningSingle' ?
-    returningSingleHeaderCopyAndContributeCopy :
-    defaultHeaderCopyAndContributeCopy;
+  const landingPageCopy = (): CountryMetaData => {
+    if (props.isReturningContributor && props.landingPageCopyReturningSinglesTestVariant === 'returningSingle') {
```
Comment (tomrf1) [inline support-frontend/assets/pages/contributions-landing/components/ContributionFormContainer.jsx]: should this be below the `US` check?

Comment (jlieb10) [reply support-frontend/assets/pages/contributions-landing/components/ContributionFormContainer.jsx]: the problem with putting it below the US one is that then I would have to modify the test to exclude the US for a period of time as well

File: support-frontend/assets/pages/contributions-landing/contributionsLandingEOY2019.scss [hunk 2]
```diff
@@ -0,0 +1,81 @@
+@import '~stylesheets/gu-sass/gu-sass';
+
+//$campaign-red: #E94F3E;
+
+.gu-content--eoy2019 {
+  //background-color: $campaign-red;
+
```
Comment (tjsilver) [inline support-frontend/assets/pages/contributions-landing/contributionsLandingEOY2019.scss]: Is this here for use in a later test?

Comment (jlieb10) [reply support-frontend/assets/pages/contributions-landing/contributionsLandingEOY2019.scss]: thanks for flagging -cleaned up

File: support-frontend/assets/pages/contributions-landing/components/ContributionFormContainer.jsx (hunk 1 above)
Comment (jlieb10) [reply support-frontend/assets/pages/contributions-landing/components/ContributionFormContainer.jsx]: per amanda, going to end this test

**Pull request merged**